app/controllers/concerns/idv_step_concern.rb

```diff
@@ -8,9 +8,21 @@ module IdvStepConcern
     before_action :confirm_idv_needed
   end
 
+  def flow_session
+    user_session['idv/doc_auth']
+  end
+
+  def pii_from_doc
+    flow_session&.[]('pii_from_doc')
+  end
+
+  # copied from doc_auth_controller
+  def flow_path
+    flow_session[:flow_path]
+  end
+
   def confirm_document_capture_complete
-    @pii = flow_session&.[]('pii_from_doc') # hash with indifferent access
-    return if @pii.present?
+    return if pii_from_doc.present?
 
     flow_path = flow_session&.[](:flow_path)
 
```

Comment:
Good catch, thanks. We do need the nil protection in case someone jumps to the /verify/ssn step without a flow_session - we were seeing 500s in prod, so we added that. I took this line out and added nil protection in `#flow_path`.